apps/patient/src/utils/general.ts

```diff
@@ -106,6 +106,23 @@ function isCloseEvent(event: types.PharmacyEvent): event is types.PharmacyCloseE
   return event.type === 'close';
 }
 
+/**
+ * Dedupes pharmacy options when merging new and existing pharmacies
+ * @param newPharmacies Pharmacies query result
+ * @param existingPharmacies Pre-loaded pharmacies
+ * @returns Deduped list of enriched pharmacy options
+ */
+export const preparePharmacyOptions = (
```

Comment:
feel like `dedupePharmacyOptions` is a better name?

Reply:
It also processes hours info